test/uk/gov/hmrc/merchandiseinbaggagefrontend/controllers/DeclarationJourneyControllerSpec.scala

```diff
@@ -94,14 +94,14 @@ trait DeclarationJourneyControllerSpec extends BaseSpecWithApplication with Core
       "no session id is set" in {
         givenADeclarationJourneyIsPersisted(startedDeclarationJourney)
 
-        val result = controller.onPageLoad(1)(buildGet(url))
+        val result = controller.onPageLoad(1, false)(buildGet(url))
```

Comment:
We agreed we'd test UI changes in selenium so we'd need extra specs. I'd expect the controller specs to shrink or become redundant in consequence.

We should test the redirect behaviour for both cases (normal navigation and change via /review-goods).